### ISSUE TITLE
Can't remove "Upgrade to OS X El Capitan" nag annoyance on OS X 10.9

### ISSUE THREAD

**Issue by MikeRich88** [jacobsalmela/NCutil#14]
I tried "NCutil.py --remove-system-center" and even that doesn't kill the vile dialog.

**Comment by jacobsalmela** [comment]
Yeah, it doesn't seem to work on Yosemite either.  I'll have to look into this.

**Comment by MikeRich88** [comment]
A little bit of additional info.

I had the “Install system data files and security updates” checkbox in the App Store pane of System Preferences, turned off, on a particular 10.9 system, for whatever reason. This is (obviously) insecure, so I turned it back on. I got the "Upgrade to OS X El Capitan" dialog less than a minute later.

So, Apple is bundling a nag screen with important security updates. Sounds more like Microsoft to me.

**Comment by jacobsalmela** [comment]
Yeah, it seems they have been doing a lot of things to prevent modifications in a non-Apple approved way (see [tccutil](https://github.com/jacobsalmela/tccutil/issues/18)).  It might just be a fruitless effort if Apple just wipes out the settings.  Not too great for enterprise users though who want to stay on a certain version of OS X.
